fontique/src/backend/fontconfig/mod.rs

```diff
@@ -90,26 +90,13 @@ impl SystemFonts {
         // First, parse the raw config files
         let mut config = Config::default();
         config::parse_config("/etc/fonts/fonts.conf".as_ref(), &mut config);
-        if let Ok(xdg_config_home) = std::env::var("XDG_CONFIG_HOME") {
-            config::parse_config(
-                PathBuf::from(xdg_config_home)
-                    .as_path()
-                    .join("fontconfig/fonts.conf")
-                    .as_path(),
-                &mut config,
-            );
-        } else if let Ok(user_home) = std::env::var("HOME") {
-            config::parse_config(
-                PathBuf::from(user_home)
-                    .as_path()
-                    .join(".config/fontconfig/fonts.conf")
-                    .as_path(),
-                &mut config,
-            );
+        let xdg_dirs = xdg::BaseDirectories::with_prefix("fontconfig").unwrap();
```

Comment:
Is unwrapping the right behaviour here? In case we couldn't determine `$HOME` previously, it looks like we just didn't parse this file?

Reply:
Removed the xdg dep